modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoverySpiMBean.java

```diff
@@ -246,11 +246,16 @@
     public long getSocketTimeout();
 
     /**
-     * Gets join timeout.
+     * Get join timeout, in milliseconds. Time to wait for joining. If node cannot connect to any address from the IP
+     * finder, the node continues to try to join during this timeout. If all addresses still do not respond, an
+     * exception will occur and the node will fail to start. If 0 is specified, it means wait forever.
```

Comment:
Get_s_ join timeout, in milliseconds. Time to wait for joining. If node cannot connect to any address from the IP
finder, the node continues to try to join during this timeout. If all addresses still do not respond, an
exception will occur and the node will fail to start. If 0 is specified, it means wait forever.